nautobot/core/jobs/bulk_actions.py

```diff
@@ -46,11 +48,25 @@ def _update_objects(self, model, form, filter_query_params, edit_all, nullified_
             filterset_cls = get_filterset_for_model(model)
 
             if filter_query_params and not filterset_cls:
-                self.logger.error(f"Filterset not found for {model}")
-                raise RunJobTaskFailed(f"Filter query provided but {model} do not have a filterset.")
+                self.logger.error(f"Filterset not found for `{model}`")
+                raise RunJobTaskFailed(f"Filter query provided but `{model}` do not have a filterset.")
```

Comment:
Should we be using `model._meta.verbose_name` in these messages instead?

```suggestion
                raise RunJobTaskFailed(f"Filter query provided but `{model}` does not have a filterset.")
```